packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -581,9 +581,13 @@ class FlutterTask extends BaseFlutterTask {
         if (dependenciesFile.exists()) {
             try {
                 // Dependencies file has Makefile syntax:
-                //   <target> <files>: <source> <files> <separated> <by> <space>
+                //   <target> <files>: <source> <files> <separated> <by> <non-escaped space>
                 String depText = dependenciesFile.text
-                return project.files(depText.split(': ')[1].split())
+                // So we split list of files by non-escaped(by backslash) space,
+                def matcher = depText.split(': ')[1] =~ /(\\ |[^\s])+/
+                // then we remove all backslashes
+                def depList = matcher.collect{it[0].replaceAll("\\\\", "")}
```

Comment:
Does `"\\\\"` seriously only remove one backslash?  That's a LOT of escaping!

Reply:
Yeah, what can you do. It feels like regex-syntax should work: `replaceAll(/\\/, "")` but somehow it doesn't.